Pipfile

```diff
@@ -4,10 +4,16 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [dev-packages]
+isort = "*"
+flake8 = "*"
+black = "*"
+autopep8 = "*"
 
 [packages]
 django = "*"
 djangorestframework = "*"
+pycodestyle = "*"
+django-money = "*"
```

Comment:
might be better to [specify the versions](https://docs.pipenv.org/en/latest/basics/#specifying-versions-of-a-package) to avoid major updates breaking the app.

Reply:
ok, I will see to it. Thank you!